### PR TITLE
 formatting: Reformat gpu_is_available() function

### LINKED ISSUE #23132: gpu_is_available

### PR DESCRIPTION
**Pull request by Boghdady9**
The function `gpu_is_available()` now correctly formatted and determines whether a GPU is available.
#

# PR Description

## Related Issue

close #23132

## Checklist

- [ x] Did you add a function?
- [ x] Did you add the tests?
- [ x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by Boghdady9** [comment]
#23215

**Comment by Boghdady9** [comment]
#23132

File: ivy/functional/ivy/device.py [hunk 1]
```diff
@@ -689,15 +689,16 @@ def gpu_is_available() -> bool:
 
     Returns
     -------
-    ret
-        Boolean, as to whether a gpu is available.
+    ret: bool
```
Comment (AnnaTz) [inline ivy/functional/ivy/device.py:692]: Hi @Boghdady9, we don't add typehints here in the docstring. You should remove this.

File: ivy/functional/ivy/device.py [hunk 2]
```diff
@@ -689,15 +689,16 @@ def gpu_is_available() -> bool:
...
+        Boolean, indicating whether a GPU is available.
 
     Examples
     --------
-    >>> print(ivy.gpu_is_available())
+    >>> print(gpu_is_available())
```
Comment (AnnaTz) [inline ivy/functional/ivy/device.py:697]: I think this was correct as it was.

File: ivy/functional/ivy/device.py [hunk 3]
```diff
@@ -689,15 +689,16 @@ def gpu_is_available() -> bool:
...
     False
     """
-    return ivy.current_backend().gpu_is_available()
+
+    return ivy.current_backend().is_gpu_available()
```
Comment (AnnaTz) [inline ivy/functional/ivy/device.py:701]: And this one as well. The way you have changed it now the name of the function is wrong.

**Review by AnnaTz — CHANGES_REQUESTED** [review]
I don't think there was any change really needed for this function to be honest.

**Comment by Boghdady9** [comment]
ok, thanks for your review. Please check my second commit @AnnaTz

**Comment by AnnaTz** [comment]
Closing this as there were no formatting changes needed.

**Comment by Boghdady9** [comment]
but why you didn’t mark it as completed @AnnaTz

**Comment by Boghdady9** [comment]
@AnnaTz 
I marked 'Allow edits by maintainer': please merge the issue

**Pull request closed without merging**